apps/code-examples/src/app/code-examples/router/href/basic/sky-href-demo.component.html

```diff
@@ -1,26 +1,33 @@
 <p>
   <span>Link where the user has access:</span>
-  <a skyHref="https://example.com">Example.com</a>
+  <a skyHref="https://example.com" data-sky-id="my-href-1">Example.com</a>
 </p>
 <p>
   <span>Link where the user has access, translated:</span>
-  <a id="allow" skyHref="allow://example.com"
+  <a skyHref="allow://example.com" data-sky-id="my-href-allow"
     >Example.com with “allow” protocol</a
   >
 </p>
 <p>
   <span>Link where the user does not have access, hidden by default:</span>
-  <a id="hidden" skyHref="deny://example.com"
+  <a skyHref="deny://example.com" data-sky-id="my-href-hidden"
     >Example.com with “deny” protocol</a
   >
 </p>
 <p>
   <span>Link where the user does not have access, shown as plain text:</span>
-  <a id="unlinked" skyHref="deny://example.com" skyHrefElse="unlink"
+  <a
+    skyHref="deny://example.com"
+    skyHrefElse="unlink"
+    data-sky-id="my-href-unlinked"
     >Example.com with “deny” protocol</a
   >
 </p>
 <p>
   <span>Link completely rewritten:</span>
-  <a skyHref="1bb-nav://spa-nav/route/in/spa?query=value">1bb-nav link</a>
+  <a
+    skyHref="1bb-nav://spa-nav/route/in/spa?query=value"
+    data-sky-id="my-href-5"
```

Comment:
Can this example be removed? Are we using it in the test?

Reply:
Removed.